### ISSUE TITLE
[dev]  ./genome/genome.fa link not created in output dir

### ISSUE THREAD

**Issue by darked89** [nf-core/chipseq#282]
Hello,

I have run:
```
nextflow run . -profile test,singularity --outdir ../out_test_nf_chipseq
```

While loading IGV session session:

```
cd ../out_test_nf_chipseq/igv/broadPeak/
igv.sh
#Loading:
igv_session.xml
``` 
IGV reported:
```
Error loading session session: Cannot locate genome: /full/path/to/out_test_nf_chipseq/genome/genome.fa
```

I have found ( in one of the subdir in ./work) and soft-linked the ```genome.fa``` to ```out_test_nf_chipseq/genome/genome.fa```
after which IGV displayed the results as expected.

This is a fairly minor nuisance, but IMHO it should be addressed. 

Many thanks for creating and maintaining such complex pipeline.

Darek Kedra

**Comment by JoseEspinosa** [comment]
Thanks for reporting @darked89, we should address it before releasing 2.0!

**Comment by JoseEspinosa** [comment]
Fixed in #279. I close this issue now but let us know if you find any other issue @darked89 Thanks!
